pkg/syncer/status/status_process.go

```diff
@@ -164,6 +164,11 @@ func (c *Controller) process(ctx context.Context, gvr schema.GroupVersionResourc
 	if !ok {
 		return fmt.Errorf("object to synchronize is expected to be Unstructured, but is %T", obj)
 	}
+	if u.GetLabels()[workloadv1alpha1.ClusterResourceStateLabelPrefix+c.syncTargetKey] == string(workloadv1alpha1.ResourceStateUpsync) {
+		logger.Info("do not update the status in upstream, since the downstreal resource is in Upsync mode")
```

Comment:
verbosity. Usually for "not doing X" we use V(3) or V(4)

Reply:
Fixed